zipline/pipeline/term.py

```diff
@@ -452,8 +453,20 @@ def _validate(self):
         if self.inputs is NotSpecified:
             raise TermInputsNotSpecified(termname=type(self).__name__)
 
-        if not self.outputs:
+        if self.outputs is NotSpecified:
+            pass
```

Comment:
Thought this `pass` might be clearer than a double negative check that says `if self.outputs is not NotSpecified`